css/layout/sidebar.css

```diff
@@ -15,8 +18,9 @@
 	flex-direction: column;
 	align-items: stretch;
 	font-weight: 500;
-	justify-content: center;
+	/* justify-content: center; */
```

Comment:
Better to remove unwanted code then commenting.

Reply:
Actually this is for my reference as there's a bug here.